### PR TITLE
test: add tests

### PR DESCRIPTION
**Pull request by ricardogobbosouza**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export interface NuxtTestContext {
   rootDir: string
   config: NuxtConfig
   nuxt: {
-    config: NuxtConfig
+    options: NuxtConfig
```
Comment (pi0) [inline src/types.ts:11]: Nice find! Maybe we should type it to `NuxtOptions` too?

File: test/generate.test.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { readFileSync } from 'fs'
+import { resolve } from 'path'
+import { setupTest, NuxtTestContext } from '../src'
+
+describe('generate', () => {
+  const ctx: NuxtTestContext = setupTest({
```
Comment (pi0) [inline test/generate.test.ts:6]: We may also destructure `{ nuxt }` for tests. Will also remove requirement of importing `NuxtTestContext` type

Comment (ricardogobbosouza) [reply test/generate.test.ts:6]: For some reason, when it desestruturar the test fails

File: src/types.ts (hunk 1 above)
Comment (ricardogobbosouza) [reply src/types.ts]: Sure

Comment (ricardogobbosouza) [reply src/types.ts]: Found, thanks to tests :smile:

File: test/generate.test.ts (hunk 2 above)
Comment (pi0) [reply test/generate.test.ts:6]: What's the bug? 😬

File: package.json [hunk 3]
```diff
@@ -34,8 +34,10 @@
     "codecov": "latest",
     "eslint": "latest",
     "husky": "latest",
+    "core-js": "2",
```
Comment (pi0) [inline package.json:37]: nuxt 2.14 should go fine with 3 too

**Pull request merged**